browser_history/__init__.py

```diff
@@ -18,8 +18,30 @@ def get_history():
         try:
             browser_object = browser_class()
             browser_output_object = browser_object.fetch()
-            output_object.entries.extend(browser_output_object.get())
+            output_object.history_entries.extend(browser_output_object.get_history())
         except AssertionError:
             utils.logger.info("%s browser is not supported", browser_class.name)
-    output_object.entries.sort()
+    output_object.history_entries.sort()
+    return output_object
+
+def get_bookmarks():
```

Comment:
There is a problem here. Even though the scope is different for `__init__.get_bookmarks` and `generic.Outputs.get_bookmarks`, for developers it would be confusing.

 Similar is the case for  `__init__.get_history` and `generic.Outputs.get_history`.

I am clueless as of now. We are lacking good names (It's a global issue xD). So, if you / @Samyak2 / Anyone else reviewing this PR has any idea for the names, Please comment.

Reply:
Even i thought of that potential issue, it would arise in the case when it is imported something like
```bash
from Outputs import get_history
```
I could do one of two things
1.  I could add a type parameter like the other functions , but that would not make the get function look good
2.  ```bash
from Outputs import get_history
```
is pretty unlikely leave it as it is